sopel/plugins/rules.py

```diff
@@ -584,9 +584,24 @@ class Rule(AbstractRule):
         <user> hello sopelunkers
         <Bot> You triggered a rule, saying hello to "sopelunkers"
 
+    Generic rules are not triggered by any specific mean, unlike commands which
```

Comment:
\*puts on Google hat\*
Did you mean:

```suggestion
    Generic rules are not triggered by any specific name, unlike commands which
```

At first I thought "means", but rules *are* triggered by specific means: text matching the pattern. Anyway.